<launch>

  <arg name="base_global_planner" default="global_planner/GlobalPlanner"/>
  <arg name="base_local_planner" default="dwa_local_planner/DWAPlannerROS"/>
  
  <param name="use_sim_time" value="false" />  
  <machine name="pmb2-5c" address="10.68.0.1" user="root" env-loader="/opt/pal/cobalt/env.sh" />
    
  <node pkg="move_base" type="move_base" name="move_base" respawn="false" output="screen" machine="pmb2-5c">
  
    <remap from="odom" to="mobile_base_controller/odom"/>    
    <remap from="cmd_vel" to="nav_vel"/>
    
    <param name="base_global_planner" value="$(arg base_global_planner)"/>
    <param name="base_local_planner" value="$(arg base_local_planner)"/>
	
    <rosparam file="$(find pmb2_lab_nav)/config/navigation/planner.yaml" command="load"/>
    <rosparam file="$(find pmb2_lab_nav)/config/navigation/recovery_behaviors.yaml" command="load"/>

    <rosparam file="$(find pmb2_lab_nav)/config/navigation/costmap_local.yaml" command="load"  />
    <rosparam file="$(find pmb2_lab_nav)/config/navigation/costmap_global.yaml" command="load" />
	
  </node>

</launch>